### ISSUE TITLE
Button 'cancel' right or left?

### ISSUE THREAD

**Issue by AnthonySendra** [kuzzleio/kuzzle-backoffice#69]
In Modal, the button 'cancel' is to the left of button submit. In create/update pages, the button 'cancel' is on the right of button submit.

Could we take a decision about this? (I don't care, but we have to choose)
(https://material.google.com/patterns/confirmation-acknowledgement.html)

**Comment by xbill82** [comment]
Yo. Very good remark.

**TL;DR**
According to my UX gurus, **there is no *best* choice**. It actually doesn't matter what we choose as long as we stay consistent with our choice. So I say, let's stick to the Material Design *Cancel-first* convention.

Norman Nielsen Group publishes [a great article exactly on this topic](https://www.nngroup.com/articles/ok-cancel-or-cancel-ok/), where they claim that

> In cases like this, it often doesn't matter what you do. Either choice has good arguments in its favor, and no choice is likely to cause usability catastrophes.

But, talking about consistency

> Deviate from the standard, and you'll easily cost users several minutes — or possibly hours — as they overlook or misuse UI elements.

And what they consider to be "the standard" is the convention adopted by the user's OS (I guess they assume the user is more exposed to the UI of the OS than the UI of the websites or applications she uses).

> Sadly, the Windows Vista User Experience Guidelines differ from the Apple Human Interface Guidelines when it comes to the sequence of OK/Cancel buttons: Windows puts OK first, Apple puts OK last

But I personally do not agree with this claim or, at least, I consider it cannot be applied to our context. Our  users are likely to be familiar with a variety of OS and probably come from the Linux world (which, in terms of UI is the less consistent).

So I propose to **stick to the Material Design Guidelines**, which (according to the link you put in your issue) [seem to adopt the *Cancel-first* convention](https://material.google.com/components/dialogs.html#dialogs-specs).

> Dismissive actions are placed directly to the left of affirmative actions and return the user to the originating screen or step in the process.

Let's make a poll. I'll post two comments to this thread, each one with a different proposals. The one that gets more votes will be adopted.

**Comment by xbill82** [comment]
Yo! Let's put Cancel first (to the left) and OK last (to the right)!

**Comment by xbill82** [comment]
Yo! Let's put OK first (to the left) and Cancel last (to the right)!

**Comment by AnthonySendra** [comment]
Okay, I'll fix this in this PR https://github.com/kuzzleio/kuzzle-backoffice/pull/65
(Cancel to the left)
